OrthoEvol/Manager/database_management.py

```diff
@@ -1,682 +1,808 @@
+        else:
+            self.database_path = Path(project_path)
+
+    def download_windowmasker_files(self, taxonomy_ids):
+        """
+        Download the WindowMasker files used in the BLAST database.
+
+        :param taxonomy_ids:  Taxonomy ids for the organisms of interest.
+        :type taxonomy_ids:  list.
+        """
+        OrthoEvolDeprecationWarning("Windowmasker files are no longer used by the most current blastx command line "
+                                    "utilities.  You can now use taxon ids directly.")
+        # <path>/<user or basic_project>/databases/NCBI/blast/db/<database_name>
+        dl_path = Path(self.database_path) / Path("NCBI") / Path('blast') / Path('windowmasker_files')
+        self.ncbiftp.getwindowmaskerfiles(taxonomy_ids=taxonomy_ids, download_path=str(dl_path))
+
+    def download_blast_database(self, database_name="refseq_rna", set_blastdb=True):
+        """
+        Download the blast database files for using NCBI's BLAST+ command line.  For other types of blast data, please
+        see the NCBIREADME.md file.
+
+        :param database_name:  A string that represents a pattern in the files of interest.
+        :type database_name:  string.
+        :param set_blastdb:  A flag that determines whether the BLASTDB environment variable is automatically set.
+        :type set_blastdb:  bool.
+        """
+        # <path>/<user or basic_project>/databases/NCBI/blast/db/<database_name>
+        dl_path = Path(self.database_path) / Path("NCBI") / Path("blast") / Path("db")
+
+        self.ncbiftp.getblastdb(database_name=database_name, download_path=str(dl_path))
+
+        env_vars = dict(os.environ).keys()
+        if set_blastdb or ("BLASTDB" not in env_vars):
+            # See if .bash_profile or .profile exists
+            bash_prof = Path("~/.bash_profile").expanduser().absolute()
+            sh_prof = Path("~/.profile").expanduser().absolute()
+            if not bash_prof.exists():
+                if not sh_prof.exists():
+                    bash_prof.touch(mode=0o700)
+                    set_prof = bash_prof
+                else:
+                    set_prof = sh_prof
+            else:
+                set_prof = bash_prof
+            self.db_mana_log.warning("Setting the PATH in %s" % str(set_prof))
+            # Use the set .*profile to append to PATH
+            with open(str(set_prof), 'r') as prof:
+                _ = prof.read()
+                bas_prof_export = "export PATH=\"%s:$PATH\"" % str(dl_path)
+                if bas_prof_export not in _:
+                    with open(str(set_prof), "a+") as b_prof:
+                        b_prof.write("export PATH=\"%s:$PATH\"" % str(dl_path))
+                    cmd = ["source %s" % str(set_prof)]
+                    stdout = self.db_mana_utils.system_cmd(cmd=cmd, stdout=sp.PIPE, stderr=sp.STDOUT, shell=True)
```

Comment:
What's happening to the stdout here?

Could it be logged?

Reply:
From `subprocess` docs:

> subprocess.PIPE¶
> Special value that can be used as the stdin, stdout or stderr argument to Popen and indicates that a pipe to the standard stream should be opened. Most useful with Popen.communicate().

It could be logged, but I will have to update this in the utils.

Reply:
Yeah, the logic is there. We can always do that before we push to master. I think next week could be a good time to do that.